config/backstage-plugins.yaml

```diff
@@ -1,11 +1,5 @@
 config:
   branches:
-    release-v1.14:
-      openShiftVersions:
-      - onDemand: true
-        version: "4.16"
-      - onDemand: true
-        version: "4.14"
```

Comment:
https://github.com/openshift-knative/hack/pull/704 should help to not clean it up yet

Reply:
Seems fixed
/unhold